### ISSUE TITLE
Add EVENTSTORE_HOME to better support running as deamon

### ISSUE THREAD

**Issue by wkruse** [EventStore/EventStore#279]
Right now starting EventStore (for example as deamon) outside of the installation directory e.g.:
```bash
exec /opt/EventStore-OSS-Linux-v3.0.0/clusternode
```
results in ```HTTP 404 Not Found``` for all web resources (the server runs fine, just doesn't find any web resources). The solution is to change the working directory:

```bash
chdir /opt/EventStore-OSS-Linux-v3.0.0
exec ./clusternode
```
It would be nice to add the requirements for running as deamon to the documentation. What about adding something like `EVENTSTORE_HOME` environment variable?

**Comment by dnauck** [comment]
Related to #553

**Comment by jen20** [comment]
How would you deal with multiple instances on one box with this?

**Comment by dnauck** [comment]
@jen20 As far as i know we'll have 2 install possibilities: a packages to unzip/untar and for Linux an extra debian package

The zip/tar packages is (always) installed in a custom path und uses resources out of its own application directory, the debian package in the system and using global systems paths for its resources (e.g. /usr/share/eventstore, /var/lib/eventstore, /var/log/eventstore, /etc/eventstore/, etc.).

The new idea in #553 will give local resources the precedence over system global resources. So even the installed packages will check: is there a local file/resources; if so use it; otherwise the normally assumed global system file/resource.

How does this help for multiple instances?

The untar/unzip packages could be unpacked in different locations (e.g. under different user accounts) and it will use its local resources with its local changes/settings -> perfectly for multi instance stuff. I've also the possibility to copy common stuff like the web-ui to /usr/share/eventstore and remove it from the local install dir to have a single resource for all instances (local resource not found, use default global location logic)

The install (debian) package does not have local resources so its always using the global default location. But i've the possibility to override this behavior by copying a modified version into the applications binary location (well, no one would do that). To run multiple instances with the install package i need to specify them in my configuration file (e.g. different NLog.config and different database path, etc.)

**Comment by pgermishuys** [comment]
This was resolved with #533 as well as https://github.com/EventStore/EventStore/commit/1536c2490b2c6856b595762fb3a1a9619eb07ae8. Please feel free to reopen if this is not resolved.
